apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss

```diff
@@ -45,6 +46,13 @@ $container-width: 309px;
   h4 {
     margin-bottom: 0.75em;
     color: $neutral_dark;
+
+    // Shows max two lines before truncating with ellipsis
+    // See https://css-tricks.com/line-clampin/
+    display: -webkit-box;
+    overflow: hidden;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
```

Comment:
all supported browsers now recognizing a `-webkit` style! what a time to be alive 😁

Reply:
Ha! Amen. I had to read that documentation over and over to make sure I was reading it right ...